test.sh

```diff
@@ -43,15 +43,11 @@ if [ ! -z "$TRAVIS" ] || [ ! -z "$GH_ACTIONS" ]; then
     cat context/instance/app.conf
   )
 fi
-set -m; ./dev-start.sh & set +m  # Without job control, I had trouble killing parent and children.
-PID=$!
-server_up 5001  # Not really needed: Cypress will wait for response.
-end-to-end/test.sh
```

Comment:
Doesn't `end-to-end/test.sh` still need to be called somewhere?

Reply:
Isn't it being called on line 75?

Reply:
Yep! Sorry to be confused.